core/quill.js

```diff
@@ -84,9 +85,6 @@ class Quill {
     if (this.options.readOnly) {
       this.disable();
     }
-    if (this.options.placeholder) {
```

Comment:
Isn't it better to make sure that the `data-placeholder` attribute is set before we set the `EDITOR_CHANGE` listener to set the `ql-blank` class which depends on this attribute?

By the way, it was [the same order](https://github.com/quilljs/quill/blob/c3247d85f0a42324d3e8aaf600d95cfbc43bdc84/core/quill.js#L81) before my initial commit.